.github/workflows/workflow.yml

```diff
@@ -126,6 +126,17 @@ jobs:
         with:
           name: basilisk_runtime.compact.compressed.wasm
           path: target/release/wbuild/basilisk-runtime/basilisk_runtime.compact.compressed.wasm
+      - name: Build benchmarking node
+        run: cargo build --release --locked --features runtime-benchmarks
+      - name: Version info
+        run: ./target/release/basilisk --version
+      - name: Run benchmarktests
```

Comment:
i think that these tests are already ran with coverage tests, but this issue is mainly about executing benchmarks itself from binary, not tests if them

Reply:
Updated. But benchmarks fail because there is a problem (session keys are not registered for validators used in the benchmark) with the collator-selection pallet (in cumulus v0.9.17). Upgrading will fix this issue. I also removed the tips pallet because we use the default weights for it and can't be benchmarked if `elections-phragmen::DesiredMembers < 2` (and is set to 1).